src/Materials/Textures/internalTexture.ts

```diff
@@ -215,10 +204,6 @@ export class InternalTexture {
     /** @hidden */
     public _compression: Nullable<string> = null;
     /** @hidden */
-    public _generateStencilBuffer: boolean = false;
-    /** @hidden */
-    public _generateDepthBuffer: boolean = false;
-    /** @hidden */
     public _comparisonFunction: number = 0;

```

Comment:
Should this still be here as well ?

Reply:
The `comparisonFunction` is not part of the RTT wrapper(?) We can have a texture which is not a RTT that has a comparison function.